docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/06_Inserting_Images.md

```diff
@@ -20,14 +20,29 @@ Image files for a website must be in either JPG, GIF or PNG format. If your imag
 
 ## Setting the alignment of an image
 
-1. Select the image and click the button **Insert from files** shown as a file icon in the HTML editor.
+1. Select the image and click the button ***Insert from files*** shown as a file icon in the HTML editor.
 2. Navigate to the ***Placement*** tab.
-3. Select the alignment for your image from the **Alignment** dropdown field.
+3. Select the alignment for your image from the ***Alignment*** dropdown field.
 * Left
 * Center
 * Left or right wrap
 4. Click the ***Insert file*** button.
 
 ## Repositioning an Image
 
-To change the position of an image that you have inserted, simply click and drag it, moving the text cursor to the position you want to move the image to. Remember, the text cursor position only defines the position of the top of the image, so you're best dragging it to the very start of a paragraph, or onto it's own line.
+To change the position of an image that you have inserted, simply click and drag it, moving the text cursor to the position you want to move the image to. Remember, the text cursor position only defines the position of the top of the image, so you're best dragging it to the very start of a paragraph, or onto its own line.
```

Comment:
The only bit that changed on this line is changing `onto it's own line` to `onto its own line`